internal/types/conditions/conditions.go

```diff
@@ -0,0 +1,14 @@
+package conditions
```

Comment:
Wanna add a status-conditions.md docs page to describe what's here?

Reply:
did it as spec docs?

Reply:
Nvm you have those values, so just need coverage for different kinds.

Reply:
Yea, I think spec docs work for now. If we add more conditions it probably deserves a page right now though it's the same thing for every kind. We can add it to every kind or list the main ones that actually reconcile in the docs?